res/css/photon-components.css

```diff
@@ -35,10 +35,43 @@
   margin-left: 8px;
 }
 
-.photon-button:hover:active {
+.photon-button:active {
   background-color: var(--grey-90-a30);
 }
 
+.photon-button-primary,
+.photon-button-default {
+  min-width: 132px;
+  height: 32px;
+  margin: 0 8px;
```

Comment:
I'm a bit surprised by this margin, I think you wanted the padding. It's already defined in `.photon-button`.